lib/src/analyzer/rules/use_alternative_class/use_alternative_class_visitor.dart

```diff
@@ -0,0 +1,61 @@
+import 'dart:collection';
+
+import 'package:analyzer/dart/ast/ast.dart';
+import 'package:analyzer_plugin/protocol/protocol_generated.dart';
+
+import '../../rule/rule_visitor.dart';
+import '../../rule/utils.dart';
+import 'use_alternative_class.dart';
+
+class UseAlternativeClassVisitor extends RecursiveRuleVisitor {
+  UseAlternativeClassVisitor({
+    required super.rule,
+    required super.result,
+    required super.config,
+  }) {
+    final allowedClasses = config.options[allowedClassesKey] as List<Object>?;
```

Comment:
What is the value for adding allowedClasses here?

Reply:
this is to override the option, but removed this option because we have no concrete use case that requires this..